app/controllers/anonymous_feedback_controller.rb

```diff
@@ -66,15 +66,21 @@ def index_params
 
   def clean_paths
     if params[:path].present?
-      params[:paths] = [params[:path]]
+      params[:paths] = params[:path]
     elsif params[:paths] && params[:paths].instance_of?(String)
-      saved_paths = Support::Requests::Anonymous::Paths.find(params[:paths])
-      params[:paths] = saved_paths.try(:paths) || params[:paths].split(',').map(&:strip)
+      params[:paths] = params[:paths].split(',').map(&:strip)
```

Comment:
The `elsif` returns a list, the `if` returns a path.  Is that right?

Reply:
Yep, good point, I've changed that back to the `if` returns a list. I'm not sure if we actually need to support the old `path` param anymore, but not confident enough to remove it.